client/components/forms/form-toggle/index.jsx

```diff
@@ -99,9 +99,11 @@ export default class FormToggle extends PureComponent {
 						aria-label={ this.props[ 'aria-label' ] }
 						tabIndex={ this.props.disabled ? -1 : 0 }
 					/>
-					<span className="form-toggle__label-content" onClick={ this.onLabelClick }>
-						{ this.props.children }
-					</span>
+					{ this.props.children && (
```

Comment:
maybe `React.Children.count( this.props.children )` instead here? Though I'm not sure if `children` can every be an empty array...

Reply:
yeah, there's some funny circumstances where children can be truthy but empty (empty array)

https://codesandbox.io/s/myp6k14k49

Don't think it blocks merging though

Reply:
Yes, the question "are `children` empty?" is an interesting one :wink:

With the straightforward `<FormToggle />`, the `children` prop won't be set at all. No problem here: there will be no label rendered ever.

The most realistic way how to get an empty array as `children` is to render a list:
```jsx
<FormToggle>{ list.map( renderItem ) }</FormToggle>
```
and then make the `list` empty. I would argue the programmer's intent was to render some label. And the code has a bug where "empty list" special case is not handled.

It's similar to passing an empty string as label:
```jsx
<FormToggle>{ '' }</FormToggle>
```
And unlike the empty array, `Children.count()` won't notice the emptiness.

With that in mind, I think that the simplest possible check that detects the `<FormToggle />` case is OK.